tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: ["src/index.ts"],
+  sourcemap: true,
+  clean: true,
+  minify: true,
+  platform: "browser",
```

Comment:
did this fix something?

Reply:
no, just copied from the react package so we have the same config. this is basically the same thing we had in the tsup cli command before in package.json before